integration_tests/test_issue_332.py

```diff
@@ -0,0 +1,13 @@
+from ltypes import i32
+
+def test_issue_332():
+    arr: i32[4]
+    arr[0] = 1
+    arr[1] = 0
+    arr[2] = -4
+    print(arr[:2])
+    print(arr[1])
+    print(arr[2])
```

Comment:
Usage of `assert` possible here to check if the outputs are actually correct?

```suggestion
    print(arr[:2])
    print(arr[1])
    print(arr[2])
    arr_2: i32[2] = arr[:2]
    assert arr_2[0] == 1
    assert arr_2[1] == 0
    assert arr[1] == 0
    assert arr[2] == -4
```

Reply:
Not yet:
```
File "/Users/thebigbool/repos/lpython/src/libasr/codegen/asr_to_llvm.cpp", line 1658
    LFORTRAN_ASSERT(ASR::is_a<ASR::Character_t>(*ASRUtils::expr_type(x.m_v)) &&
AssertFailed: ASR::is_a<ASR::Character_t>(*ASRUtils::expr_type(x.m_v)) && n_dims == 0
```

Reply:
Makes sense. Can you try fixing the bug?

Reply:
I think the LLVM backend currently only supports string-slicing assignments and not arrays because they can be n-dimensional. Is that intentional?